api/paymentMethodsApi.ts

```diff
@@ -249,8 +249,10 @@ export class PaymentMethodsApi {
      * @param country Filters the results to only the items which support this country code. A country is formatted as 2-letter ISO country code.
      * @param currency Filters the results to only the items which support this currency code. A currency is formatted as 3-letter ISO currency code.
      * @param environment Filters the results to only the items available in this environment.
+     * @param limit Defines the maximum number of items to return for this request.
+     * @param cursor A cursor that identifies the page of results to return. This is used to paginate the results of this API.  For the first page of results, this parameter can be left out. For additional pages, use the value returned by the API in the &#x60;next_cursor&#x60; field. Similarly the &#x60;previous_cursor&#x60; can be used to reverse backwards in the list.
      */
-    public async listBuyerPaymentMethods (buyerId?: string, buyerExternalIdentifier?: string, country?: string, currency?: string, environment?: 'development' | 'staging' | 'production', options: {headers: {[name: string]: string}} = {headers: {}}) : Promise<{ response: http.IncomingMessage; body: PaymentMethodsTokenized;  }> {
+    public async listBuyerPaymentMethods (buyerId?: string, buyerExternalIdentifier?: string, country?: string, currency?: string, environment?: 'development' | 'staging' | 'production', limit?: number, cursor?: string, options: {headers: {[name: string]: string}} = {headers: {}}) : Promise<{ response: http.IncomingMessage; body: PaymentMethodsTokenized;  }> {
```

Comment:
@cbetta Might be better to look at using objects as argument as the changing position will break existing contracts.